recipe/meta.yaml

```diff
@@ -25,12 +26,12 @@ requirements:
   host:
     - python
     - pip
-    - sip
+    - sip >=4.19
     - qt
     - pyqt
   run:
     - python
-    - sip
+    - sip >=4.19
     - qt
     - pyqt
```

Comment:
You should probably wait for https://github.com/conda-forge/pyqt-feedstock/pull/50